src/index.js

```diff
@@ -1,26 +1,33 @@
 'use strict';
 
-var Retriever = require('./retriever');
-var RequestFactory = require('./request-factory');
+var assign = require('lodash.assign');
+var ResultList = require('./result-list');
+
+var _defaults = {
+  page: 1,
+  per_page: 20,
+  safesearch: true,
+  url: 'https://pixabay.com/api'
+};
 
 var pixabayjs = {
+  _auth: {},
+
   authenticate: function(username, key) {
-    this.username = username;
-    this.key = key;
+    this._auth.username = username;
+    this._auth.key = key;
   },
 
-  defaults: function(defaults) {
-    this.defaults = defaults;
+  get defaults () {
+    return _defaults;
+  },
+  set defaults (value) {
+    _defaults = assign(_defaults, value);
```

Comment:
with vanilla getter/setter if they set `defaults` to `{page: 2}` we(they) lose `per_page`, `url`, and `safesearch`